manifests/t/tastyworks/tastyworks/2.3.1/tastyworks.tastyworks.locale.en-US.yaml

```diff
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.2.8 $debug=QUSU.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: tastyworks.tastyworks
+PackageVersion: 2.3.1
+PackageLocale: en-US
+Publisher: tastyworks # This is intentionally wrong to allow for automatic redirection to tastyworks.tastytrade
+# PublisherUrl:
+# PublisherSupportUrl:
+# PrivacyUrl:
+# Author:
+PackageName: tastyworks # This is intentionally wrong to allow for automatic redirection to tastyworks.tastytrade
```

Comment:
Is this like a tombstone, to facilitate redirection?

Reply:
Yes; Its described in the FAQ